apps/backend/src/datasources/postgres/CallDataSource.test.ts

```diff
@@ -9,10 +9,13 @@ import { Proposal } from '../../models/Proposal';
 import { dummyCallFactory } from '../mockups/CallDataSource';
 import CallDataSource from './CallDataSource';
 import database from './database';
-import { createCallObject, createProposalObject } from './records';
+import { CallRecord, createCallObject, createProposalObject } from './records';
 
 const callDataSource = new CallDataSource();
 
+const tomorrow = new Date(new Date().setDate(new Date().getDate() + 1));
+const yesterday = new Date(new Date().setDate(new Date().getDate() - 1));
+
```

Comment:
While it is a bit longer, this version may be easier to understand. No strong preference from my side.
```suggestion
const today = new Date();
const tomorrow = new Date(today);
tomorrow.setDate(tomorrow.getDate() + 1);

const yesterday = new Date(today);
yesterday.setDate(yesterday.getDate() - 1);


```